packages/botonic-core/index.d.ts

```diff
@@ -80,6 +85,7 @@ export class HandOffBuilder {
   withAgentEmail(email: string): HandOffBuilder
   withNoteURL(note: string): HandOffBuilder
   withCaseInfoURL(caseInfo: string): HandOffBuilder
+  withShadowing(shadowing: boolean): HandOffBuilder
```

Comment:
withShadowing(shadowing **= true**): HandOffBuilder

Reply:
if I do this typescript complains: `A parameter initializer is only allowed in a function or constructor implementation.`

Reply:
that's true, my bad. It should be:
`withShadowing(shadowing?:boolean): HandOffBuilder`